packages/host/ember-cli-build.js

```diff
@@ -47,6 +47,9 @@ module.exports = function (defaults) {
             }),
           ],
           resolve: {
+            alias: {
+              dompurify: 'dompurify/dist/purify.es.js',
```

Comment:
it also might be the case that this alias is effecting how the matrix service is able to load the dom purify module. in which case you'll need to adjust the https://github.com/cardstack/boxel/blob/8e3f3f92c2f5cbc29b4c89bc6576f732858865c9/packages/host/app/services/matrix-service.ts#L19 to import dom purify correctly

Reply:
Fixed https://github.com/cardstack/boxel/pull/371/commits/8f1a760e6c4cc514ccc9605359dfdc8f91241c4c